.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+omit =
+    hil/ext/obm/impi.py
+    hil/ext/switches/brocade.py
+    hil/ext/switches/_console.py
+    hil/ext/switches/edllnos9.py
```

Comment:
Typo... And I still don't understand why these files are skipped

Reply:
Because we don't use these in travis, because they are drivers for hardware that travis doesn't have access to. So the coverage will always be 0%, and it's just noise.

Reply:
Sorry, I still don't understand. For two (interconnected) reasons, actually, I'm confused.

1. There seem to be files in tests/unit/ which correspond to these skipped drivers. So are there not unit tests?
2. If there really aren't unit tests for those files, then don't you deserve that 0% anyway?

I think I must be missing something obvious here...

Reply:
@jeremyfreudberg, we do have tests, but we don't run them in travis because they require physical hardware that isn't accessible from travis -- they are hardware drivers after all. We try to be good about running them manually when we change stuff pertinent to the hardware.

Reply:
(There are a couple tests that mention some of the drivers under `tests/unit/`, but those are migrations and stuff for the client library; they don't really poke the functionality of those modules. The relevent tests are under `deployment/`).

Reply:
Fair enough. I think it's okay to omit these from the coverage calculation.

(I did the quickest of pass-throughs over the skipped driver files... there's nothing there that couldn't have a unit test if we actually wanted to. So like I said earlier, we could say that the lack of tests does mean 0% coverage. But really a lot of the unit tests that could end up being written would just be the `assert_called_with` over and over, which is arguably not useful... so I'm fine with the way things are now, at least in the context of this PR.)